node/node.go

```diff
@@ -758,7 +758,8 @@ func (n *Node) ConfigureRPC() (*rpccore.Environment, error) {
 		Config: *n.config.RPC,
 	}
 	if err := rpcCoreEnv.InitGenesisChunks(); err != nil {
-		return nil, err
+		errMsg := "could not create the genesis file chunks and cache them: %s"
```

Comment:
What's the benefit of having a separate variable?

Reply:
It's just for readability. There's no performance benefit if that's what you are asking.

Reply:
The opposite approach (direct embedding) is as readable. But it's up to you!